File: add/codesnippet/CSharp/e3b03beb-cdf1-41be-a632-_1.cs

	private void validateUserEntry()
	{

		// Checks the value of the text.

		if(serverName.Text.Length == 0)
		{

			// Initializes the variables to pass to the MessageBox.Show method.

			string message = "You did not enter a server name. Cancel this operation?";
                        string caption = "Error Detected in Input";
			MessageBoxButtons buttons = MessageBoxButtons.YesNo;
			DialogResult result;

			// Displays the MessageBox.

			result = MessageBox.Show(message, caption, buttons);

			if (result == System.Windows.Forms.DialogResult.Yes)
			{

				// Closes the parent form.

				this.Close();

			}

		}

	}

File: add/codesnippet/VisualBasic/e3b03beb-cdf1-41be-a632-_1.vb
    Private Sub ValidateUserEntry()

        ' Checks the value of the text.

        If ServerName.Text.Length = 0 Then

            ' Initializes variables to pass to the MessageBox.Show method.

            Dim Message As String = "You did not enter a server name. Cancel this operation?"
            Dim Caption As String = "Error Detected in Input"
            Dim Buttons As MessageBoxButtons = MessageBoxButtons.YesNo

            Dim Result As DialogResult

            'Displays the MessageBox

            Result = MessageBox.Show(Message, Caption, Buttons)

            ' Gets the result of the MessageBox display.

            If Result = System.Windows.Forms.DialogResult.Yes Then

                ' Closes the parent form.

                Me.Close()

            End If

        End If

    End Sub